lib/locales/ko.js

```diff
@@ -33,12 +33,12 @@ module.exports = {
         'Orbital rotation': '궤도 수정',
         'Pan': '이동',
         'Produced with Plotly': 'Plotly 제공',
-        'Reset': 리셋',
-        'Reset axes': '축 리셋',
-        'Reset camera to default': 'camera를 기본값으로 리셋',
-        'Reset camera to last save': 'camera를 마지막으로 저장한 값으로 리셋',
-        'Reset view': 'view 리셋',
-        'Reset views': 'views 리셋',
+        'Reset': 초기화',
```

Comment:
I think this line should be 
```
'Reset': '초기화',
```
Now missing a start quotation mark after the key.
@JJByun would you please fix that so that tests pass on the CircleCI?
Thanks!